File: 4-median-of-two-sorted-arrays/median-of-two-sorted-arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        std::vector<int> mergedVec = nums1;
        mergedVec.insert(mergedVec.end(), nums2.begin(), nums2.end());

        std::sort(mergedVec.begin(), mergedVec.end());
        int n = mergedVec.size();
        if(n % 2 == 0)
            return (mergedVec[n/2-1] + mergedVec[n/2]) / 2.0;
        else
            return mergedVec[n / 2];
       
    }
};

File: 4-median-of-two-sorted-arrays/median-of-two-sorted-arrays.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        newArray = nums1 + nums2
        newArray = sorted(newArray)
        if newArray == []:
            return null
        else:
            if len(newArray) % 2 == 0:
                return ((newArray[len(newArray)//2-1] + newArray[len(newArray)//2]) / 2)
            else:
                return newArray[len(newArray)//2]
        